app/views/IndexView.scala.html

```diff
@@ -16,7 +16,8 @@
 
```

Comment:
I get a boat load of "unusedImport" warnings during a compile of the project with no real reason (sbt just seems to consider comments inside '_.scala.html' files as unused imports), and I've seen in the build.sbt file we've got the silencer plugin which has the option to suppress these but will likely suppress other warnings as well... Is this something that we are going to implement or do we move forward knowing that there will be irrelevant warnings (not errors mind you!) in the startup log?

Reply:
I'll checkout the branch and do a `clean compile` and see if this is something we can resolve.

Reply:
Sorted.